src/rules/require-sql-template.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+
+import assert from 'assert';
+import flora from 'flora-sql-parser';
+import get from 'lodash.get';
+
+/**
+ * Instances.
+ */
+
+const parser = new flora.Parser();
+
+/**
+ * Check if `literal` is an SQL query.
+ */
+
+function isSqlQuery(literal) {
+  if (!literal) {
+    return false;
+  }
+
+  try {
+    parser.parse(literal);
```

Comment:
I couldn't think of a better way to make this actually work. Unless we assume everything gets inlined in some kind of `query()` call, we loose control over assignments.